### ISSUE TITLE
fermata + whole note interaction

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#622]
High whole notes cause problems for fermatas:

<img width="586" alt="screen shot 2017-06-13 at 11 14 48 pm" src="https://user-images.githubusercontent.com/3487289/27104974-2530bdca-508e-11e7-8fee-85bdd19541f1.png">

```
**kern
1ff;
1gg;
1aa;
1bb;
1ccc;
1ddd;
1eee;
1fff;
==
*-
```
and breves:

<img width="591" alt="screen shot 2017-06-13 at 11 17 22 pm" src="https://user-images.githubusercontent.com/3487289/27105070-80b5733e-508e-11e7-9736-c068fa376e5e.png">

(the problem occurs one note higher in breves compared to whole notes)

```
**kern
0ff;
0gg;
0aa;
0bb;
0ccc;
0ddd;
0eee;
0fff;
==
*-
```

But not for quarter notes:

<img width="478" alt="screen shot 2017-06-13 at 11 15 59 pm" src="https://user-images.githubusercontent.com/3487289/27105010-481d569a-508e-11e7-8083-0ab86e4403f9.png">

```
**kern
4ff;
4gg;
4aa;
4bb;
4ccc;
4ddd;
4eee;
4fff;
==
*-
```

Or half notes:

<img width="520" alt="screen shot 2017-06-13 at 11 16 28 pm" src="https://user-images.githubusercontent.com/3487289/27105051-66849c10-508e-11e7-87a4-0da25869fdda.png">

```
**kern
2ff;
2gg;
2aa;
2bb;
2ccc;
2ddd;
2eee;
2fff;
==
*-
```

The problem most likely occurs because the fermata thinks that it can fit underneath the high stemless notes and be more compactly spaced with the staff.  But fermatas should not be allowed to sneak under the notehead/chord that they are attached to.

**Comment by donbyrd** [comment]
I wonder if upside-down fermatas behave the same way.

**Comment by craigsapp** [comment]
Yes they do:

<img width="595" alt="screen shot 2017-06-14 at 7 45 44 am" src="https://user-images.githubusercontent.com/3487289/27117147-88ced528-50d5-11e7-9573-ba50cb887f9c.png">

```
**kern
1e;<
1d;<
1c;<
1B;<
1A;<
1G;<
1F;<
1E;<
==
*-
!!!RDF**kern: < = below
```

**Comment by rettinghaus** [comment]
It happens with all floating elements and a stem is the only salvation. If you shorten the stem same happens on all notes. 

![floating_object](https://user-images.githubusercontent.com/7693447/27119408-1c4a1ec0-50e0-11e7-9589-cdc07cc8ee45.png)

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
  <meiHead>
    <fileDesc>
      <titleStmt>
        <title />
      </titleStmt>
    </fileDesc>
  </meiHead>
  <music>
    <body>
      <mdiv>
        <score>
          <scoreDef>
            <staffGrp>
              <staffDef clef.shape="C" clef.line="1" meter.count="4" meter.unit="4" n="1" lines="5" />
            </staffGrp>
          </scoreDef>
          <section>
            <measure>
              <staff n="1">
                <layer n="1">
                  <note xml:id="note-with-fermata-01" oct="7" pname="f" dur="4"/>
                  <note xml:id="note-with-fermata-02" oct="7" pname="f" dur="4" stem.len="8"/>
                  <note xml:id="note-with-turn" oct="7" pname="c" dur="4" stem.len="5"/>
                  <note xml:id="note-with-mordent" oct="7" pname="d" dur="4" stem.len="6"/>
                </layer>
              </staff>
              <fermata startid="#note-with-fermata-01"/>
              <fermata startid="#note-with-fermata-02"/>
              <turn startid="#note-with-turn"/>
              <mordent startid="#note-with-mordent"/>
            </measure>
          </section>
        </score>
      </mdiv>
    </body>
  </music>
</mei>
```

This is a nasty bug and should be fixed with the breath-mark-placing-bug together (for 1.1.1).

**Comment by lpugin** [comment]
Fixed with a8b56bd58e506928703e6f186ce38d571364d5b9

**Comment by rettinghaus** [comment]
much nicer, thanks!

![floating_object](https://user-images.githubusercontent.com/7693447/27125291-217252a0-50f4-11e7-8a2a-ef5ec06a5700.png)
